### PR TITLE
created the first block of the project

### PR DESCRIPTION
**Pull request by Natali-Pchelkina**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (ghost) [inline Triphouse-project/index.html:1]: All images should be in Triphouse-project folder.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <title>What do we offer</title>
+</head>
+<body>
+    <div class="wrapper">
+        <h2>What do we offer</h2>
+        <div class="wrapper-line">
+        </div>
+        <div class="wrapper-offer">
+            <div class="wrapper-item">
+                <img src="../img/Vector.jpg" alt="headphones" width="75px" height="91.67px">
```
Comment (ghost) [inline Triphouse-project/index.html]: You should write width and hight only in css. And add the same height and width for all images.

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                <p>Support 24/7</p>
+            </div>
+            <div class="wrapper-item">
+                <img src="../img/Vector (1).jpg" alt="message" width="83.33px" height="83.33px">
```
Comment (ghost) [inline Triphouse-project/index.html]: See the previous comment.

File: Triphouse-project/index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                <p>Communicate directly</p>
+            </div>
+            <div class="wrapper-item">
+                <img src="../img/Vector (2).jpg" alt="connectijn" width="100px" height="66.67px">
```
Comment (ghost) [inline Triphouse-project/index.html]: See the previous comment.

File: Triphouse-project/index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                <p>Book online</p>
+            </div>
+            <div class="wrapper-item">
+                <img src="../img/Vector (3).jpg" alt="thumb up" width="91.67px" height="83.33px">
```
Comment (ghost) [inline Triphouse-project/index.html]: See the previous comment.

File: Triphouse-project/index.html (hunk 1 above)
Comment (Natali-Pchelkina) [reply Triphouse-project/index.html:1]: moved the pictures to the Triphouse-project folder

File: Triphouse-project/index.html (hunk 2 above)
Comment (Natali-Pchelkina) [reply Triphouse-project/index.html]: registered the height and width of the pictures in the css file, set the same dimensions for all pictures

File: Triphouse-project/index.html (hunk 3 above)
Comment (Natali-Pchelkina) [reply Triphouse-project/index.html]: registered the height and width of the pictures in the css file, set the same dimensions for all pictures

File: Triphouse-project/index.html (hunk 4 above)
Comment (Natali-Pchelkina) [reply Triphouse-project/index.html]: registered the height and width of the pictures in the css file, set the same dimensions for all pictures

File: Triphouse-project/index.html (hunk 5 above)
Comment (Natali-Pchelkina) [reply Triphouse-project/index.html]: registered the height and width of the pictures in the css file, set the same dimensions for all pictures

**Pull request merged**